src/foam/nanos/dig/DIG.js

```diff
@@ -53,6 +53,25 @@ foam.CLASS({
     },
     'cmd',
     'format',
+{
+      name: 'digColumns',
```

Comment:
I would just call this `columns` or maybe `fields`
I'd also start with making it a StringArray. We might want to change that later to something else but I think StringArray is  a good place to start.